src/components/ActivityDetails.js

```diff
@@ -0,0 +1,190 @@
+import { getPlaybookRun, getPlaybookRunSystems } from '../actions';
+import './Status.scss';
+
+export const normalizeStatus = (status) => ({
+    running: 'running',
+    pending: 'running',
+    failure: 'failure',
+    canceled: 'failure',
+    success: 'success',
+    acked: 'success'
+})[status];
+
+export const renderStatus = (status, text) => ({
+    running: <Text component={ TextVariants.p } >
+        <InProgressIcon
+            className="ins-c-remediations-running"
+            aria-label="connection status" />
+        { text || 'Running' }
+    </Text >,
+    success: <Text component={ TextVariants.p }>
+        <CheckCircleIcon
+            className="ins-c-remediations-success"
+            aria-label="connection status" />
+        { text || 'Success' }
+    </Text>,
+    failure: <Text component={ TextVariants.p }>
+        <TimesCircleIcon
+            className="ins-c-remediations-failure"
+            aria-label="connection status" />
+        { text || 'Failed' }
+    </Text>
+})[status];
+
+const statusText = (executorStatus) => ({
+    running: <Text component={ TextVariants.p } >
+        Running
+    </Text >,
+    success: <Text className="ins-c-remediations-success" component={ TextVariants.p }>
+        Suceeded
+    </Text>,
+    failure: <Text className="ins-c-remediations-failure" component={ TextVariants.p }>
+        Failed
+    </Text>
+})[executorStatus];
+
+const statusSummary = (executorStatus, systemsStatus) => {
+    console.log(executorStatus, systemsStatus);
+    return <Split style={ { display: 'flex' } } className="ins-c-remediations-status-bar">
+        <SplitItem>
+            { statusText('success') }
+        </SplitItem>
+        <SplitItem>
+            { renderStatus('success', systemsStatus.success) }
+        </SplitItem>
+        <SplitItem>
+            { renderStatus('failure', systemsStatus.failure) }
+        </SplitItem>
+        <SplitItem>
+            { renderStatus('running', systemsStatus.running) }
+        </SplitItem>
+    </Split>;
+};
+
+const ActivityDetail = ({
+    remediation,
+    playbookRun,
+    playbookRunSystems,
+    getPlaybookRun,
+    getPlaybookRunSystems
+}) => {
+    useEffect(() => {
+        getPlaybookRun();
+        getPlaybookRunSystems();
+
+    }, []);
+
+    // const systemsStatus = playbookRunSystems.reduce((acc, { status }) => ({ ...acc, [normalizeStatus(status)]: acc[normalizeStatus(status)] + 1 })
+    //     , {  running: 0, success: 0, failure: 0 });
+    const systemsStatus = { running: 1, success: 2, failure: 1 };
+
+    return (
+        <React.Fragment>
+            <PageHeader>
+                <Breadcrumb>
+                    <BreadcrumbItem>
+                        <Link to={ `/${remediation.id}` }> { remediation.name } </Link>
+                    </BreadcrumbItem>
+                    <BreadcrumbItem isActive> <DateFormat type='exact' date={ playbookRun.data.created_at } /> </BreadcrumbItem>
```

Comment:
The format does not match the mocks. https://github.com/RedHatInsights/insights-remediations-frontend/blob/35a1476d6bea26cea69f2cd856ae6a6ceb52345c/src/Utilities/model.js#L40 should be used instead

Reply:
I am using the `DateFormat`component that should be aligned with https://www.patternfly.org/v4/design-guidelines/content/grammar-and-terminology#date-and-time In the mock there are days and months switched. If it is not super important, I would stick to using the same standard format.

Reply:
EDIT: I have noticed that there shouldn't be displayed seconds, I will make a PR to Frontend components to fix it.

Reply:
@PanSpagetka we should be aligned with https://docs.google.com/document/d/1hivSfe48717VhlLRtjyqReS5twbSLuPC6MISJt3MTG4/edit